### PR TITLE
Mention supported network path, default values for optional fields and link to custom API source creation

### PR DESCRIPTION
**Pull request by praveen-elastic**
The goal of this PR is to address the following changes suggested in issue #15 :
- Mention the type of network path supported by the connector.
- Mention the default values taken by the connector for optional fields.
- Add a link to elastic documentation for creating a custom API source.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,7 +65,9 @@ The steps above are relevant to all users. Some users may require additional fea
 Collect the information that is required to connect to your network drives:
 
 - The domain name where the network drive is present.
-- The network drives path the connector will crawl to fetch files.
+- The network drives path the connector will crawl to fetch files. It should be the name of the folder shared via SMB followed by the path relative to that shared folder
+
+  (For Example: If `Users` is the name of SMB shared folder and folder to be fetched is `folder1`, then the path would be mentioned as `Users/folder1`)
```
Comment (leemthompo) [inline README.md:70]:
```suggestion
- The network drives path the connector will crawl to fetch files. This is the name of the folder shared via SMB, followed by the path relative to that shared folder. For example: if the name of the of SMB shared folder is `Users` and the folder to be fetched is `folder1`, then the path is `Users/folder1`).
```

Comment (leemthompo) [reply README.md:70]: Minor language tidy up

File: README.md [hunk 2]
```diff
@@ -105,9 +107,10 @@ Each network drives connector syncs data from network drives into a Workplace Se
 
 Create a content source within Kibana:
 
-1. Navigate to **Enterprise Search** → **Workplace Search** → **Sources** → **Add Source** → **Custom Content Source**.
+1. Navigate to **Enterprise Search** → **Workplace Search** → **Sources** → **Add an organization content source** → **Custom API Source**.
 2. Name your Content Source, (e.g. Network Drives Connector).
-3. Choose **Configure network drives Connector**.
+
+For more details please refer [Elastic Documentation for creating a custom content source](https://www.elastic.co/guide/en/workplace-search/current/workplace-search-custom-api-sources.html#create-custom-source).
```
Comment (leemthompo) [inline README.md:113]:
```suggestion
For more details see [Elastic Documentation for creating a custom content source](https://www.elastic.co/guide/en/workplace-search/current/workplace-search-custom-api-sources.html#create-custom-source).
```

File: README.md [hunk 3]
```diff
@@ -533,6 +536,8 @@ enterprise_search.host_url: https://my-deployment.ent.europe-west1.gcp.cloud.es.
 
 Whether the connector should sync [document-level permissions (DLP)](#use-document-level-permissions-dlp) from network drives.
 
+By default, it is set to `Yes` i.e. by default, the connector will try to sync document-level permissions.
```
Comment (leemthompo) [inline README.md:539]:
```suggestion
Set to `Yes` by default i.e. the connector will try to sync document-level permissions.
```

**Review by leemthompo — COMMENTED** [review]
I made some very minor language clean up suggestions. 

I'll let @ppf2 give formal approval as he created the issue.

File: README.md [hunk 4]
```diff
@@ -65,7 +65,7 @@ The steps above are relevant to all users. Some users may require additional fea
 Collect the information that is required to connect to your network drives:
 
 - The domain name where the network drive is present.
-- The network drives path the connector will crawl to fetch files.
+- The network drives path the connector will crawl to fetch files. This is the name of the folder shared via SMB, followed by the path relative to that shared folder. For example: if the name of the of SMB shared folder is `Users` and the folder to be fetched is `folder1`, then the path is `Users/folder1`.
```
[the remark this answers is not in the document]
Comment (praveen-elastic) [reply README.md:68]: The connector supports SMB1 and SMB2. Adding this in README

File: README.md [hunk 5]
```diff
@@ -76,6 +76,8 @@ Collect the information that is required to connect to your network drives:
 
 Later, you will [configure the connector](#configure-the-connector) with these values.
 
+ℹ️ The connector uses [pysmb](https://pysmb.readthedocs.io/en/latest/) module for connecting to your Network Drives. pysmb supports SMB1 and SMB2.
```
Comment (leemthompo) [inline README.md:79]:
```suggestion
ℹ️ The connector uses the [pysmb](https://pysmb.readthedocs.io/en/latest/) module for connecting to your Network Drives. `pysmb` supports SMB1 and SMB2.
```

**Review by leemthompo — APPROVED** [review]
👍

**Pull request merged**